module/input_handler.py

```diff
@@ -257,13 +257,13 @@ def handle_input(self, input_words: list) -> None:
         while input_words:
             word = input_words.pop(0).strip()
 
-            if word == "&&":
+            if word == "&&" and len(command) > 0:
```

Comment:
this should probably not be two separate checks for `len(command) > 0`, but just one single check